lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
```

Comment:
@marcotuts I just realized that this input has no ```<label>``` element associated with it.  Ideally, I like to see visible labels.  They persist (unlike placeholder text) which helps lowvision and those with a cognitive impairment.  However, sometimes that doesn't work with the UI.  Acceptable options are an offscreen label or aria-label if you can't make a visual label work.   Given the fact that we have placeholder text and an icon button, I'm OK with offscreen or aria-label here.